neo/SmartContract/InteropService.cs

```diff
@@ -147,8 +147,7 @@ internal static bool CheckWitness(ApplicationEngine engine, UInt160 hash)
             {
                 Cosigner usage = tx.Cosigners.FirstOrDefault(p => p.Account.Equals(hash));
                 if (usage is null) return false;
-                if (usage.Scopes.HasFlag(WitnessScope.Global))
-                    return true;
+                if (usage.Scopes == WitnessScope.Global) return true;
```

Comment:
Could we revert this @erikzhang @shargon? I think it's better having Global as flag 0x80, combinable with others in a OR logic (which is the "standard" behavior). Otherwise it will be difficult to explain how these flags work, which is bad for security. Let's adopt a plain OR logic, where everything combined with Global is Global.

Reply:
I don't understand what does it mean when `Global` is a combinable flag.

Reply:
Anything OR Global should mean Global. Meaning that, if someone sets a flag to Global, regardless of any other flags, it should be global. I think this has a strong (and correct) meaning. Global is the most open of all, so combining with it, just make things more open. The AND logic would be more strange, because Global would be "absorved" by any other flags, narrowing it down.

However... if we don't set any flag, what the expected behavior.. global? In this case, we should consider again letting it be 0x00, and never combining it :joy: